exercises/exercise4/checkSpecialCases.xquery

```diff
@@ -0,0 +1,149 @@
+xquery version "1.0";
+
+declare namespace local = "local";
```

Comment:
declare a module namespace instead of local: module namespace csc= "kalahMancala/checkSpecialCases";